doc/data/messages/m/missing-final-newline/good.py

```diff
@@ -1 +1,9 @@
-# This is a placeholder for correct code for this message.
+# using LF
+
+foo(bar, baz) #  \n
+print(123)  # \nEOF
+
+# using CRLF
+
+foo(bar, baz)  # \r\n
```

Comment:
```suggestion
eat("apple", "candy")  # \r\n
```

Reply:
Oh, yes that makes sense! Should I replace `print(123)` too?

Reply:
No it's ok, school children generally knows about numbers under 4 😛